portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/models/blob/DigitalDocument.java

```diff
@@ -0,0 +1,57 @@
+package org.datatransferproject.types.transfer.models.blob;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.datatransferproject.types.transfer.models.DataModel;
+
+/**
+ * This is intended to by a sub set of schema.org's DigitalDocument
+ * (see: https://schema.org/DigitalDocument).  Additional fields from the schema.org spec should
```

Comment:
nice. I assume you can't import schema.org classes?

also, WDYT about separating the schema.org spec into its own class and add our fields in a wrapper?